SITnett/templates/produksjoner/forestilling_fjern.html

```diff
@@ -7,9 +7,9 @@ <h3>Er du sikker på at du vil fjerne forestillingen av<br>
 
 	<form action="{% url 'forestilling_fjern' forestilling.id %}" method="post">
 		{% csrf_token %}
-		<input type="submit" value="ja">
+		<input type="submit" value="Ja">
 	</form>
-	<a href="{% url 'produksjon_info' forestilling.produksjon.id %}">nei</a>
+	<a href="{% url 'produksjon_info' forestilling.produksjon.id %}">Nei</a>
```

Comment:
Tilsvarende som for medlem, vi kan vurdere en advarsel om at dette ikke kan angres. Om vi ønsker at det _skal_ kunne angres, kan vi legge inn en `published: true` attributt på elementer i databasen, og så heller sette denne til `false` når ting "slettes". Det er jo mulig det ikke er helt stuerent på brukere, men på forestillinger burde det gå greit.

Reply:
Jeg legger inn advarsler.